src/IO/S3/Client.cpp

```diff
@@ -681,6 +705,9 @@ Client::doRequestWithRetryNetworkErrors(RequestType & request, RequestFn request
         std::exception_ptr last_exception = nullptr;
         for (Int64 attempt_no = 0; attempt_no < max_attempts; ++attempt_no)
         {
+            if (!isClientForDisk())
+                incrementProfileEvents<IsReadMethod>(ProfileEvents::S3ReadRequestAttempts, ProfileEvents::S3WriteRequestAttempts);
```

Comment:
counters for `isClientForDisk()` == `true` are not needed for now; this PR aims to debug server-side throttling in backups.